tests/BUILD.bazel

```diff
@@ -30,11 +30,19 @@ java_fuzz_target_test(
     ],
 )
 
+java_binary(
+    name = "HookDependenciesFuzzerHooks",
```

Comment:
I think that it still has a chance to work, but it may have very confusing effects for users if they suddenly find some of their other classes on the bootstrap class path. AFAIK this could have happened before as well, so maybe has never been possible for "real" projects with fuzz test jars consisting of more than a single class.